server/fishtest/templates/tests_live_elo.mak

```diff
@@ -6,7 +6,12 @@
 </script>
 
 <div class="container">
-  <h2>Live Elo for SPRT test <a href="/tests/view/${str(run['_id'])}">${str(run['_id'])}</a></h2>
+  <h2>
+    <center>
```

Comment:
Hello Stéphane, The `<center>` tag has been deprecated in HTML 4 and been removed from HTML5 in favor of the CSS text-align property, which can be applied to the <div> element, but by doing so the <a> tag (username) is alone centered and other fields (i.e Commit, Info, Submitter etc.) are not